packages/api/src/external/carequality/command/cq-directory/list-cq-directory.ts

```diff
@@ -2,7 +2,7 @@ import { OrganizationWithId } from "@metriport/carequality-sdk/client/carequalit
 import { out } from "@metriport/core/util/log";
 import { makeCarequalityManagementApiOrFail } from "../../api";
 
-const BATCH_SIZE = 100;
+const BATCH_SIZE = 1_000;
```

Comment:
Noticed this was too low. The rebuild uses 5,000 items per page, so this is totally fine and preferred to load lots of records.